orca-queue/src/test/kotlin/com/netflix/spinnaker/orca/q/handler/CompleteExecutionHandlerTest.kt

```diff
@@ -205,18 +205,6 @@ object CompleteExecutionHandlerTest : SubjectSpek<CompleteExecutionHandler>({
         subject.handle(message)
       }
 
-      it("updates the pipeline status") {
-        verify(repository).updateStatus(PIPELINE, pipeline.id, stageStatus)
-      }
-
-      it("publishes an event") {
-        verify(publisher).publishEvent(check<ExecutionComplete> {
-          assertThat(it.executionType).isEqualTo(pipeline.type)
-          assertThat(it.executionId).isEqualTo(pipeline.id)
-          assertThat(it.status).isEqualTo(stageStatus)
-        })
```

Comment:
Is there a corresponding test that it _doesn't_ update the status if there _are_ still other stages doing stuffs.

Reply:
There is a `verifyNoMoreInteractions(queue)` in here, I'll add

Reply:
```kotlin
it("does not update execution status") {
  verifyNoMoreInteractions(repository)
}
```